<ui version="4.0" >
 <class>BIGHOME_UI_commentsManagerClass</class>
 <widget class="QWidget" name="BIGHOME_UI_commentsManagerClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>39</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>BIGHOME_UI_commentsManager</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="star1" >
       <property name="cursor" >
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text" >
        <string>star1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="star2" >
       <property name="cursor" >
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text" >
        <string>star2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="star3" >
       <property name="cursor" >
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text" >
        <string>star3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="star4" >
       <property name="cursor" >
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text" >
        <string>star4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="star5" >
       <property name="cursor" >
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text" >
        <string>star5</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
